zipline/assets/asset_db_schema.py

```diff
@@ -162,6 +163,29 @@
     sa.Column('asset_type', sa.Text),
 )
 
+supplementary_mappings = sa.Table(
+    'supplementary_mappings',
+    metadata,
+    sa.Column(
+        'id',
```

Comment:
I included it since we also have it in the `equity_symbol_mappings` table. It's populated with the index of the dataframe of mappings that we write, which ends up just being the default dataframe index, since we don't set it to a particular column. Do you think it's unnecessary?